Notepad.py

```diff
@@ -53,11 +53,22 @@
 red_icon=tk.PhotoImage(file='icons2/red.png')
 monokai_icon=tk.PhotoImage(file='icons2/monokai.png')
 night_blue_icon=tk.PhotoImage(file='icons2/night_blue.png')
+lavender_icon=tk.PhotoImage(file='icons2/lavender.png')

```

Comment:
It's working fine, but the icons have white background look different than previously present

Reply:
Change the icons then it's reday to merge